app/helpers/govuk_rails_compatibile_link_helper.rb

```diff
@@ -0,0 +1,157 @@
+require "html_attributes_utils"
+
+module GovukRailsCompatibileLinkHelper
+  using HTMLAttributesUtils
+
+  def govuk_link_classes(*styles, default_class: "#{brand}-link")
+    if (invalid_styles = (styles - link_styles.keys)) && invalid_styles.any?
+      fail(ArgumentError, "invalid styles #{invalid_styles.to_sentence}. Valid styles are #{link_styles.keys.to_sentence}")
+    end
+
+    [default_class] + link_styles.values_at(*styles).compact
+  end
+
+  def govuk_button_classes(*styles, default_class: "#{brand}-button")
+    if (invalid_styles = (styles - button_styles.keys)) && invalid_styles.any?
+      fail(ArgumentError, "invalid styles #{invalid_styles.to_sentence}. Valid styles are #{button_styles.keys.to_sentence}")
+    end
+
+    [default_class] + button_styles.values_at(*styles).compact
+  end
```

Comment:
Should these be private methods? Can't think of why you'd use them outside of a link or button helper.

If not, maybe move them below the other methods, and add a brief comment to explain what they do? It threw me a bit seeing these alongside the more obvious helper methods below.

Reply:
So these are exposed to make it [easier for devs to use Rails' _other_ link helpers](https://govuk-components.netlify.app/helpers/link/#class-helpers), like `link_to_if` and `link_to_unless`, `link_to_unless_current`, etc.